README.md

```diff
@@ -206,7 +206,7 @@ To simplify development, e.g. via the `bin/console` described above, there is a
 You can use it to query almost every object and relation in Netbox.
 
 ```bash
-cat dump.sql | docker-compose exec postgres psql -U postgres
+docker exec -i netbox-client-ruby_postgres_1 psql -U postgres < dump.sql
```

Comment:
`netbox-client-ruby_postgres_1` could change. What about `docker-compose exec postgres psql -U postgres  < dump.sql`?

Reply:
That throws an error: 
```$ docker-compose exec postgres psql -U postgres < dump.sql
the input device is not a TTY
ERROR: 1
```
And `docker-compose exec` doesn't have a -i switch like `docker exec` has

Reply:
👍🏼